snorkel/analysis/metrics.py

```diff
@@ -208,7 +178,6 @@ def _preprocess(gold, pred, prob, ignore_in_gold, ignore_in_pred):
     "recall": recall_score,
     "f1": f1_score,
     "fbeta": fbeta_score,
-    "roc-auc": roc_auc_score,
```

Comment:
Why removing?

Reply:
Restored in a future commit when we wrap scikit-learn